action.yml

```diff
@@ -25,6 +17,34 @@ inputs:
     description: "Additional arguments for stack invocations"
     required: true
     default: ""
+  stack-query-arguments:
+    description: "Additional arguments in `stack query` invocations."
+    required: true
+    default: ""
+  stack-path-arguments:
+    description: "Additional arguments in `stack path` invocations."
+    required: true
+    default: ""
+  stack-setup-arguments:
+    description: "Additional arguments in `stack setup` invocations."
+    required: true
+    default: ""
+  stack-build-arguments:
+    description: "Additional arguments in `stack build` invocations."
```

Comment:
```suggestion
    description: "Additional arguments for all `stack build` invocations."
```

Reply:
tried to fix this in https://github.com/freckle/stack-action/pull/35/commits/8de998704248d960d0d6944c3c4212fff4e3ce91